client/src/components/List.jsx

```diff
@@ -6,7 +6,7 @@ const List = (props) => {
   const { list } = props;
   return (
     <div className={styles.backgroundDiv}>
-      {list.map(element => <Comment key={element._id} comment={element} />)}
+      {list.map(element => <Comment comment={element} />)}
```

Comment:
Let's add this key property back in, because otherwise React will keep throwing that warning about keys in an array. Probably best to combine the _id with the comment index.